addons/web/static/src/legacy/js/widgets/model_field_selector_popover.js

```diff
@@ -150,7 +150,7 @@ var ModelFieldSelectorPopover = Widget.extend({
      * @returns {Object}
      */
     getSelectedField: function () {
-        return _.findWhere(this.pages[this.chain.length - 1], {name: _.last(this.chain)});
+        return _.findWhere(this.pages[this.chain.length - 1], {name: this.chain.slice(-1)[0]});
```

Comment:
While you change this line, maybe also change:
```suggestion
        return this.pages[this.chain.length - 1].find(({ name }) => name === this.chain.at(-1);
```

Reply:
Because i'm a robot ... i only do one thing at a time

Reply:
(haha, because I do function by function)

Reply:
And findWhere is in my next PR ( already done )